snap/snapcraft.yaml

```diff
@@ -791,8 +788,6 @@ parts:
       - -usr/lib/*
       - -usr/share/doc/libnorm1/*
       - -usr/share/doc/libpgm-5.2-0/*
-      - -usr/share/doc/libsodium23/*
-      - -usr/share/doc/libzmq5/*
```

Comment:
It is best to keep the exclusion lists (denoted by the `-` prefix) for app-service-configurable and kuiper parts. Those parts pull the files from the snaps which still have zmq. The exclusion lists are to avoid pulling in identical files from multiple locations, which result in build errors. If the files to exclude don't exist, it will not throw any error.

The app-service-configurable and kuiper apps bundles in this snap have been deprecated and we'll just remove them altogether soon.

Reply:
Can be ignored since the build is passing right now.